go.mod

```diff
@@ -1,21 +1,24 @@
 module github.com/charmbracelet/huh
 
-go 1.18
+go 1.20
```

Comment:
Is there any particular reason to bump this up to Go 1.20?

Reply:
Nope, downgrading...